consul_config.py.ctmpl

```diff
@@ -68,6 +68,7 @@ INCOMING_EXCHANGE = '''{{template "KEY" "incoming_exchange"}}'''
 INCOMING_QUEUE = '''{{template "KEY" "incoming_queue"}}'''
 UNIQUE_EXCHANGE = '''{{template "KEY" "unique_exchange"}}'''
 UNIQUE_QUEUE = '''{{template "KEY" "unique_queue"}}'''
+FOLLOW_LIST_QUEUE = '''{{template "KEY" "follow_list_queue"}}'''
```

Comment:
Probably need to add `WEBSOCKETS_SERVER_URL` to this file too.

Reply:
I think we can leave this to the production fixes as well.